src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -673,7 +675,7 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
         }
 
         people = this._groupPeople || [];
-      } else if (input) {
+      } else if (!this.groupId && input) {
```

Comment:
`!this.groupId` will always be true here since you are checking for it above. It can be removed

Reply:
Thanks! 
2f7b3d6